frontend/src/scenes/teamLogic.tsx

```diff
@@ -2,9 +2,13 @@ import { kea } from 'kea'
 import api from 'lib/api'
 import { teamLogicType } from './teamLogicType'
 import { TeamType } from '~/types'
+import { userLogic } from './userLogic'
 import { toast } from 'react-toastify'
 
 export const teamLogic = kea<teamLogicType<TeamType>>({
+    connect: {
+        actions: [userLogic, ['loadUser']],
```

Comment:
These connect lines are not needed, you can simply call the logic below (as you have) directly.